tests/ui/swap.fixed

```diff
@@ -187,8 +188,16 @@ const fn issue_9864(mut u: u32) -> u32 {
     u + v
 }
 
-#[allow(clippy::let_and_return)]
+macro_rules! issue_10421 {
```

Comment:
When testing external macros, we need to use auxiliary crates. So this should be move to auxiliary crates like this: https://github.com/rust-lang/rust-clippy/pull/9467/files#diff-aaffce1df73745edd6634ef9afcde3925f4f782b8ac7da721c6d5b0846571ae9

This docs can also be helpful: https://rustc-dev-guide.rust-lang.org/tests/compiletest.html#building-auxiliary-crates

Reply:
Fixed